rotkehlchen/chain/ethereum/modules/eth2/utils.py

```diff
@@ -137,6 +137,16 @@ def scrape_validator_daily_stats(
                 column_pos += 1
             elif column_pos == 2:
                 pnl = _parse_fval(column.string, 'income')
+                # if the validator makes profit in the first day beaconchain returns a
+                # profit of deposit + validation reward. We need to substract the deposit value
+                # to obtain the actual pnl.
+                # Example: https://beaconcha.in/validator/999/stats
+                if pnl > ONE:
```

Comment:
Kinda nitpicky since the current implementation will probably always work, but I think it will be cleaner to check if the value is greater than 32 and then subtract 32 from the pnl

Reply:
This was my first approach but then I tough that you might miss to commit a block and then you will get penalized, in that case you balance on first day would be < 32 ETH and I guess that the API will have the same issue

Reply:
Alright then :+1:

Reply:
you don't commit a block. Back then it was only attestations. And maybe some empty blocks. But yes can be zero or negative if you got slashed or if your validator just does not come offline.

But this in general looks like a flawed approach.

Why not deal with day of activation? So just find the first date and do this for the first date in the export?

Reply:
Ping on this one and the comments around this as pinged you in discord.